### ISSUE TITLE
Range does not accept only one argument

### ISSUE THREAD

**Issue by GoogleCodeExporter** [philhassey/tinypy#2]
```
If one runs `print(range(10))`, tinypython will raise an:

Exception:
tp_get: KeyError: 0
```

Original issue reported on code.google.com by `Bailey.D.R@gmail.com` on 5 Apr 2008 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
In current tinypy, range() can only accept restrict three arguments.
```

Original comment by `ybc2084@gmail.com` on 22 Apr 2008 at 5:17

**Comment by GoogleCodeExporter** [comment]
```
I fixed this issue at the same time as issue #5.  Please see the patch in issue 
#5.
```

Original comment by `dwhall...@gmail.com` on 25 Apr 2008 at 5:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `philhas...@gmail.com` on 26 Apr 2008 at 12:12
- Changed state: **Verified**
